src/Core/src/Handlers/ProgressBar/ProgressBarHandler.Windows.cs

```diff
@@ -4,36 +4,43 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ProgressBarHandler : ViewHandler<IProgress, ProgressBar>
+	public partial class ProgressBarHandler : ViewHandler<IProgress, Grid>
```

Comment:
Why does this fix it? Why does ProgressBar require an additional container?

If this is the correct way to fix it, then it needs some comments to explain it, since it's different than most of the handlers.

Reply:
Tried to force Layout etc but for now, to render the native Windows ProgressBar require to wrap it in a Container. We can review it with the WinUI Team and explore other ways. For now, added a comment.